source/MaterialXTest/ShaderValid.cpp

```diff
@@ -562,38 +430,49 @@ TEST_CASE("GLSL MaterialX documents", "[shadervalid]")
     mx::ShaderGeneratorPtr shaderGenerator = mx::GlslShaderGenerator::create();
     shaderGenerator->registerSourceCodeSearchPath(searchPath);
 
-    mx::FilePath path = mx::FilePath::getCurrentPath() / mx::FilePath("documents/Tests");
+    mx::FilePath path = mx::FilePath::getCurrentPath() / mx::FilePath("documents/TestSuite");
     mx::StringVec dirs;
     std::string baseDirectory = path;
-    getSubDirectories(baseDirectory, dirs);
+    mx::getSubDirectories(baseDirectory, dirs);
     for (auto dir : dirs)
     {
         mx::StringVec files;
         mx::getDocumentsInDirectory(dir, files);
         for (std::string file : files)
         {
-            mx::FilePath filePath = path / mx::FilePath(file);
-            std::string filename = filePath;
-            log << "MTLX Filename: " << filename << std::endl;
             mx::DocumentPtr doc = mx::createDocument();
+            loadLibraries({ "stdlib", "sxpbrlib" }, searchPath, doc);
+
+            mx::FilePath filePath = mx::FilePath(dir) / mx::FilePath(file);
+            std::string filename = filePath;
             readFromXmlFile(doc, filename);
 
             std::vector<mx::NodeGraphPtr> nodeGraphs = doc->getNodeGraphs();
-            std::vector<mx::OutputPtr> outputList;
-            for (mx::NodeGraphPtr nodeGraph : nodeGraphs)
+            std::vector<mx::OutputPtr> outputList = doc->getOutputs();
+            // TO ADD: Checking for shaders to validate
+            if (!nodeGraphs.empty() || !outputList.empty())
             {
-                log << "NodeGraph: " << nodeGraph->getName() << std::endl;
-                std::vector<mx::OutputPtr> nodeGraphOutputs = nodeGraph->getOutputs();
-                for (mx::OutputPtr output : nodeGraphOutputs)
+                log << "MTLX Filename: " << filename << std::endl;
```

Comment:
Move this hear to avoid dumping out for all include files, also removed other labels since the validation test dumps out element information already.